apps/blog/content/posts/what-is-memebattle.en.mdx

```diff
@@ -7,8 +7,106 @@ tags: ['MemeBattle']
+Meanwhile, we periodically returned to the original idea, but didn't create real product 😢
+
+<InstagramPost publicationId="BaRdqExl35J" description="A post shared by Themezv (@artem_zverev.9)" />
+
+In parallel with hackathons we have done some education projects. For example, in union with other students, organized IT community AVTF – few meetups by different streams (web development, mobile development, databases, information security, data science). Just so you know, on the first meetup were more than 150 people!
+
+<Image src='/content-images/what-is-memebattle/ITC2.jpg' alt="Photo from first meetup of IT community AVTF" width={1920} height={925} />
+
+We often discussed about MemeBattle and told to friend. But we could never answer the question "What is MemeBattle". Usually we respond like:
+> MemeBattle – is not a dot on a map. MemeBattle – is people.
+
+<InstagramPost publicationId="BuPCRKrlI4C" description="A post shared by MemeBattle (@memebattle.world)" />
+
+Looking back, I can answer:
+><b>MemeBattle</b> – small community around few technical projects, where we studied new technologies and approaches. <br /> Also this is the team with which we participated in hackathons and other activities.
+
+## MemeBattle today
+
+The initial concept of an application centered around battle of memes is a thing of the past.
+
+At the moment MemeBattle is primarily an <b>open community</b> united by common projects.
+We are open to new ideas and projects, also we invite everyone to join us.
+
+There are few projects in development by MemeBattle today.
+
+### Ligretto
+
+<b>Ligretto</b> – dynamic online multiplayer game, where needed to put cards as soon as possible and outperform opponents with reaction and dexterity.
+
+url: [https://ligretto.app/](https://ligretto.app/)
+
+Play with friends anytime – one round takes only few minutes. Our colleagues spend a free time between meetings sometimes)
+
+### GameHub
+
+<b>GameHub</b> – online games service. Unfortunately there is only one game so far. But in the ideal future it will be the portal of online games in one ecosystem.
+
+url: [https://mems.fun/](https://mems.fun/)
+
+Our lives
+We are witnessing a profound transformation in people's lives. Whe world is becoming global and remote.
+In the past, we used to have the chance to effortlessly gather with friends or colleagues at a party or in the office, enjoy ourselves, and engage in board games.
+
+Nowadays, many companies are spread across the globe, making it more difficult to gather around one table.
+GameHub and games within it can facilitate spending time together by playing.
+
+### Blog
+
+This blog. It's not just MemeBattle team news, but
+
+There will be not only news from the MemeBattle team, but also various posts from everyone.
+First of all, would like to share technical discoveries and experience. For example, one of next posts planned as description of using variable fonts research in this blog.
```

Comment:
First of all, we would like to share technical discoveries and experiences. For example, one of the following posts is planned as a description of using variable fonts research in this blog.